recipe/meta.yaml

```diff
@@ -101,11 +76,11 @@ outputs:
         - python >=3.6
         - virtualenv
         - wheel
-        - zlib 1.2.11
+        - zlib
         # coin or deps
         - coin-or-utils
         - coin-or-osi
-        - coin-or-clp =1.17.4
+        - coin-or-clp
```

Comment:
```suggestion
        - coin-or-clp =1.17.4
```
Lets try

Reply:
If I remember correctly versions 1.17.5 and 1.17.6 were broken with or-tools which is why this is introduced. And you're right, it should've been added to run as well.

Reply:
> If I remember correctly versions 1.17.5 and 1.17.6 were broken with or-tools which is why this is introduced.

What about the current 1.17.7?

Reply:
> What about the current 1.17.7?

That wasn't available at the time, so not sure. That's why I would like to ask Ted first.

Reply:
Actually, since he's not available and you're also the maintainer of most these packages, should we remove it and see what happens? @h-vetinari

Reply:
Yeah, feel free to remove it from my POV